test/run_test.py

```diff
@@ -232,6 +241,9 @@ def test_distributed(executable, test_module, test_directory, options):
     'cuda_primary_ctx': test_cuda_primary_ctx,
     'cpp_extensions': test_cpp_extensions,
     'distributed': test_distributed,
+    'ge_config_simple' : test_jit_ge,
```

Comment:
Using a curried function here, rather than string processing, would be much clearer.